website/content/docs/k8s/operations/gossip-encryption-key-rotation.mdx

```diff
@@ -0,0 +1,91 @@
+---
+layout: docs
+page_title: Gossip Encryption Key Rotation
+description: Rotate the Gossip Encryption Key on Kubernetes Cluster safely
+---
+
+# Rotating Gossip Encryption Key
+
+The following instructions provides a step-by-step manual process for rotating gossip encryption keys on Consul Servers that are deployed onto a Kubernetes cluster with Consul on Kubernetes.
```

Comment:
Should link to something that describes what gossip encryption keys are

Reply:
>on Consul Servers

Don't we also need to rotate clients?

Reply:
sigh! i mis-wrote it. I wanted to say perform these operation on the consul server pod, but i dont think that actually matters. Let me correct that.